### ISSUE TITLE
Question for m1 max GPU performance

### ISSUE THREAD

**Issue by h4rk8s** [divamgupta/stable-diffusion-tensorflow#28]
prompt:a bit cat 

steps: 50
size: 512 * 512
cfg: 7.5
number: 1

pytorch: 36.77s / 1.2~1.4it/s

tf: 42.95s / 1.1 ~ 1.4 it/s

WHY ?

**Comment by h4rk8s** [comment]
M1 Max 16inch 64GB, enable high performance model.The fan rotation is about 2000～3000

**Comment by AlexanderPolyakov** [comment]
I have similar setup (10CPU/24GPU, 32GB) and also see performance on par with pytorch implementation with ~1.4it/s.

**Comment by h4rk8s** [comment]
> I have similar setup (10CPU/24GPU, 32GB) and also see performance on par with pytorch implementation with ~1.4it/s.

yes, same settings,  if using lstein/stable-diffusion it works 28s/~1.7it/s (with enable CFPGAN)
